src/app/client/src/app/modules/public/module/offline/services/content-manager/content-manager.service.ts

```diff
@@ -36,7 +36,6 @@ export class ContentManagerService {
   updateContentDownloadStatus(contentDownloadList) {
     this.contentDownloadStatus = {};
     _.forEach(contentDownloadList, content => {
-      if (content.addedUsing === 'download') {
```

Comment:
WHy we remove this condition ?

Reply:
Because it saves the status of downloaded content only, not for imported content. So, download status is not getting shown on the course toc page when we import the content.